packages/react-native/Libraries/StyleSheet/StyleSheetTypes.d.ts

```diff
@@ -27,6 +27,8 @@ export type DimensionValue =
 type AnimatableNumericValue = number | Animated.AnimatedNode;
 type AnimatableStringValue = string | Animated.AnimatedNode;
 
+export type CursorValue = 'auto' | 'pointer';
```

Comment:
I could easily add the rest of the web / w3c spec pointer values here. Desktop uses them and potentially Android. For this PR though, it seemed better to limit.